poecalc.py

```diff
@@ -17,13 +17,15 @@
 def root(request):
 	return Response.render(request, 'index.jinja2', {})
 
-def analyze_auras(request, account, character):
+def analyze_auras(request, account, character, aura_effect=None):
 	# eventlet encodes PATH_INFO as latin1
 	# https://github.com/eventlet/eventlet/blob/890f320b/eventlet/wsgi.py#L690
 	# because PEP-0333 says so https://github.com/eventlet/eventlet/pull/497
 	account = account.encode('latin1').decode('utf-8')
 	character = character.encode('latin1').decode('utf-8')
-	results, vaal_results = aura_analyzer.analyze(account, character)
+	if aura_effect:
+		aura_effect = int(aura_effect.encode('latin1').decode('utf-8'))
```

Comment:
that seems a little overkill for a number
```suggestion
		aura_effect = int(aura_effect)
```